gc/include/omrmm.hdf

```diff
@@ -227,6 +227,7 @@ typedef uintptr_t (*condYieldFromGCFunctionPtr) (OMR_VMThread *omrVMThread, uint
 		<data type="uintptr_t" name="regionSize" description="heap region size" />
 		<data type="uintptr_t" name="regionCount" description="total heap region count" />
 		<data type="uintptr_t" name="arrayletLeafSize" description="arraylet leaf size" />
+		<data type="uintptr_t" name="isArrayletDoubleMappingEnabled" description="set to true if arraylet double mapping status is enabled, false otherwise" />
```

Comment:
Since we are changing the signature of the hook, we can potentially break downstream projects (OpenJ9), until they update the use of the hook.
So, let's create the clone this hook. Don't change the original. Only _V2 version will have the extra argument.
Once downstream projects get updated, we'll remove the original hook.